core/services/gateway/handler_factory.go

```diff
@@ -9,15 +9,15 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
 	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/config"
 	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers/capabilities"
 	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers/functions"
-	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers/webapicapabilities"
 	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/network"
 )
 
 const (
-	FunctionsHandlerType   HandlerType = "functions"
-	DummyHandlerType       HandlerType = "dummy"
-	WebAPICapabilitiesType HandlerType = "web-api-capabilities"
+	FunctionsHandlerType HandlerType = "functions"
+	DummyHandlerType     HandlerType = "dummy"
+	CapabilitiesType     HandlerType = "capabilities"
```

Comment:
What is the consequence of making this change? Will this break existing functionality?